atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/config/TargetedSweepRuntimeConfig.java

```diff
@@ -53,7 +53,7 @@ void checkShardSize() {
 
     @Value.Default
     public long pauseMillis() {
-        return 500L;
+        return 5L;
```

Comment:
Let's do 50 until we're confident we need to go lower. I'm wary of the load increase from always being this fast. If we need to lower the pause this much, we may need to invest in smarter backoffs instead (continuing if the last iteration did work, otherwise pausing for longer).